com.civicrm.hrjobroles/views/include/add_new_role.html

```diff
@@ -99,11 +99,8 @@
                         </ui-select-choices>
                       </ui-select>
                     </div>
-                    <a href="/civicrm/admin/options/hrjc_location?reset=1"
-                       class="crm-option-edit-link medium-popup crm-hover-button"
-                       target="_blank" title="Edit Locations"
-                       data-option-edit-path="civicrm/admin/options/hrjc_location?reset=1">
-                      <i class="crm-i fa-wrench"></i>
+                    <a class="pointer">
+                      <i class="crm-i fa-wrench" ng-click="jobroles.openOptionsEditor('/civicrm/admin/options/hrjc_location?reset=1', 'hrjc_location')"></i>
```

Comment:
If you are OK with removing URL param from the function, please make sure to update here as well.